Added PyYAML

bastianeicher opened the pull request:
New line for `python/index.html`:

```html
      <li><a href='pyyaml'>PyYAML</a> (YAML parser and emitter for Python)</li></ul>
```

talex5 commented:
Doesn't work:

```
Command failed: tar: 3.12: Not found in archive
tar: Exiting with failure status due to previous errors
... extracting archive with: tar -xf /tmp/0install-b85a2c-download -C /root/.cache/0install.net/implementations/0install-unpack-3956dc51 --no-same-owner --no-same-permissions -z 3.12
... unpacking archive 'https://github.com/yaml/pyyaml/archive/3.12.tar.gz'
... downloading implementation /mnt/pyyaml.xml 3.12 (id=sha1new=da39a3ee5e6b4b0d3255bfef95601890afd80709)
```

bastianeicher commented:
Sorry about that. During "development" of the feed I switched the download URL from `https://api.github.com/repos/yaml/pyyaml/zipball/{version}` (which has top-level folders named `{version}`) to
`https://github.com/yaml/pyyaml/archive/{version}.tar.gz` (which has top-level folders named `pyyaml-{original}`) and missed the last implementation when updating. Fixed now.
